### DISCUSSION TITLE
Aborted called on a running task

### DISCUSSION THREAD

**Discussion by RJ4100** [uNetworking/uWebSockets.js#1059] [Q&A] [answered]
Is there a way to prevent aborted being called for a specific call? I want to wait until the function completes. I see this example:   if (!res.aborted) {
    res.cork(() => {
      res.end(r);
    });
  } 

However, I need a way to respond to the user when the res is done or aborted. I see the request aborts in 3 seconds. I'm looking to increase timeout to like 4 seconds.

**Comment by uNetworkingAB** [comment]
> prevent aborted being called

If the peer disconnects, which can happen at any time, you technically have no way to respond. You can't prevent this.

**Comment by uNetworkingAB** [comment, marked as answer]
"Aborted called on a running task" <- this caption is good. I understand what you mean. But there is no technical way to prevent this, you need to either let your running task complete and then discard it if aborted, or support prematurely breaking the running task. Either solution will work.

**Reply by RJ4100** [reply]
Ok so there is no default timeout uWebSockets.js HTTP request? The abort is caused by the client timing out, correct?

**Reply by uNetworkingAB** [reply]
Abort is for when the request is no longer valid, for any reason. There is no timeout server side but your web browser may timeout if it wants to.

**Reply by RJ4100** [reply]
Yes I confirm it's client side. Thanks for the quick reply.
